test/check-application

```diff
@@ -167,7 +167,7 @@ class TestNavigator(testlib.MachineCase):
             b.key_press(["\b"] * 12)
 
             # Copy item path
-            b.click("button:contains('not-hidden')")
+            b.click("[data-item='not-hidden']")
```

Comment:
This is a separate commit; it looks unrelated, so that is okay. But again, there should be a "why" and some explanation why it failed before. In this case it's obvious that the old code is broken ("not-hidden" is not a human visible string), but how did that ever work before?

Reply:
Split it into a separate PR :+1:

Reply:
FTR, my concern was not that the fix was in this PR, but that it wasn't properly justified. Anyway, approved the other one.